xchange-core/src/main/java/org/knowm/xchange/instrument/FuturesContract.java

```diff
@@ -0,0 +1,119 @@
+package org.knowm.xchange.instrument;
+
+import java.io.Serializable;
+
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.utils.jackson.InstrumentDeserializer;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+/**
+ * Base object for financial instruments supported in xchange such as CurrencyPair, Future or Option
+ */
+@JsonDeserialize(using = InstrumentDeserializer.class)
+public class FuturesContract extends Instrument
```

Comment:
Please check class org.knowm.xchange.derivative.FuturesContract. This one duplicates it.

Reply:
done